# HG changeset patch
# User alta88@fixall.com
# Date 1717609449 -7200
# Parent  48e80dae1e9c09af132ab6ced955cb4237cc32d0
QFB fixes for TB changes in 128:
1. Tab stop on clear button
2. Adjust css for clear/search buttons

diff --git a/mail/base/content/widgets/search-bar.inc.xhtml b/mail/base/content/widgets/search-bar.inc.xhtml
--- a/mail/base/content/widgets/search-bar.inc.xhtml
+++ b/mail/base/content/widgets/search-bar.inc.xhtml
@@ -10,10 +10,9 @@
     <button id="clear-button"
             type="reset"
             class="button button-flat icon-button"
             data-l10n-id="search-bar-clear-button"
-            hidden="hidden"
-            tabindex="-1">
+            hidden="hidden">
       <slot name="clear-button"></slot>
     </button>
     <button id="search-button"
             class="button button-flat icon-button"
diff --git a/mail/components/storybook/stories/search-bar.stories.mjs b/mail/components/storybook/stories/search-bar.stories.mjs
--- a/mail/components/storybook/stories/search-bar.stories.mjs
+++ b/mail/components/storybook/stories/search-bar.stories.mjs
@@ -34,9 +34,8 @@ const Template = ({ label, disabled }) =
         type="reset"
         data-l10n-id="search-bar-clear-button"
         class="button button-flat icon-button"
         hidden="hidden"
-        tabindex="-1"
       >
         <slot name="clear-button"></slot>
       </button>
       <button id="search-button" class="button button-flat icon-button">
diff --git a/mail/themes/shared/mail/search-bar.css b/mail/themes/shared/mail/search-bar.css
--- a/mail/themes/shared/mail/search-bar.css
+++ b/mail/themes/shared/mail/search-bar.css
@@ -49,26 +49,26 @@ input {
   flex-direction: column;
   justify-content: space-around;
 
   &.button-flat.icon-button {
-    padding: 4px;
-    margin: 0;
+    padding: 2px;
 
     &:focus-visible {
-      outline-offset: -1px;
+      outline-style: solid;
+      outline-width: 2px;
+      outline-color: var(--search-focus-outline-color);
+      outline-offset: var(--search-outline-offset);
+      background-color: var(--search-bar-focus-background);
 
       @media (-moz-windows-accent-color-in-titlebar) {
         outline-offset: -4px;
       }
     }
   }
 
   #clear-button& {
-    height: 22px;
     /* Width of the search button. */
     inset-inline-end: 26px;
-    margin-block: auto;
-    padding: 2px;
   }
 }
 
 div {
